macos/snd_.ksy

```diff
@@ -0,0 +1,201 @@
+        type: u4
+  sound_command:
+    instances:
+      cmd:
+        value: raw_cmd&0x7FFF
+        enum: cmd_type
+      is_data_offset:
+        value: raw_cmd&0x8000
+      sound_header_type:
+        pos: param2 + 20
+        type: u1
+        enum: sound_header_type
+        if: is_data_offset > 0 and cmd == cmd_type::buffer_cmd
+      sound_header:
+        pos: param2
+        size-eos: true
+        if: is_data_offset > 0 and cmd == cmd_type::buffer_cmd
+        type:
+          switch-on: sound_header_type
+          cases:
+            sound_header_type::standard: standard_sound_header
+            sound_header_type::extended: extended_sound_header
+            sound_header_type::compressed: compressed_sound_header
+    seq:
+      - id: raw_cmd
+        type: u2
+      - id: param1
+        type: u2
+      - id: param2
+        type: u4
+  standard_sound_header:
+    seq:
+      - id: sample_ptr
+        type: u4
+      - id: lenght
+        type: u4
+      - id: sample_rate
+        type: u2
+      - id: sample_rate_frac
+        type: u2
+      - id: loop_start
+        type: u4
+      - id: loop_end
+        type: u4
+      - id: encode
+        type: u1
+        enum: sound_header_type
+      - id: base_frequency
+        type: u1
+      - id: sample_area
+        size: lenght
+  extended_sound_header:
+    seq:
+      - id: sample_ptr
+        type: u4
+      - id: num_channels
+        type: u4
+      - id: sample_rate
+        type: u2
+      - id: sample_rate_frac
+        type: u2
+      - id: loop_start
+        type: u4
+      - id: loop_end
+        type: u4
+      - id: encode
+        type: u1
+        enum: sound_header_type
+      - id: base_frequency
+        type: u1
+      - id: num_frames
+        type: u4
+      - id: aiff_sample_rate
+        size: 10
+        #this is a float80
+      - id: marker_chunk
+        type: u4
+      - id: instrument_chunks
+        type: u4
+      - id: aes_recording
+        type: u4
+      - id: sample_size
+        type: u2
+      - id: future_use_1
+        type: u2
+      - id: future_use_2
+        type: u4
+      - id: future_use_3
+        type: u4
+      - id: future_use_4
+        type: u4
+      - id: sample_area
+        size: num_frames * num_channels * sample_size / 8
+  compressed_sound_header:
+    seq:
+      - id: sample_ptr
+        type: u4
+      - id: num_channels
+        type: u4
+      - id: sample_rate
+        type: u2
+      - id: sample_rate_frac
+        type: u2
+      - id: loop_start
+        type: u4
+      - id: loop_end
+        type: u4
+      - id: encode
+        type: u1
+        enum: sound_header_type
+      - id: base_frequency
+        type: u1
+      - id: num_frames
+        type: u4
+      - id: aiff_sample_rate
+        size: 10
+        #this is a float80
+      - id: marker_chunk
+        type: u4
+      - id: format
+        size: 4
+        type: str
+        encoding: ASCII
+      - id: future_use_2
+        type: u4
```

Comment:
As above, this should be called `reserved`.

Reply:
Done.